lib/puppet/provider/package/apt.rb

```diff
@@ -14,7 +14,7 @@
   commands :aptcache => "/usr/bin/apt-cache"
   commands :preseed => "/usr/bin/debconf-set-selections"
 
-  defaultfor :operatingsystem => [:debian, :ubuntu]
+  defaultfor :operatingsystem => [:debian, :ubuntu, :CumulusLinux]
```

Comment:
I actually think that would be better but was a bit cautious in making a
more radical change. Should I submit an updated PR?

Sent from the road while on tour.
On Mar 31, 2015 8:16 PM, "Matthaus Owens" <notifications@github.com> wrote:

> In lib/puppet/provider/package/apt.rb
> <https://github.com/puppetlabs/puppet/pull/3778#discussion_r27535558>:
>
> > @@ -14,7 +14,7 @@
> >    commands :aptcache => "/usr/bin/apt-cache"
> >    commands :preseed => "/usr/bin/debconf-set-selections"
> >
> > -  defaultfor :operatingsystem => [:debian, :ubuntu]
> > +  defaultfor :operatingsystem => [:debian, :ubuntu, :CumulusLinux]
>
> Any reason we can't just default on :osfamily => [:debian]?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/puppetlabs/puppet/pull/3778/files#r27535558>.
>